gradle-scripts/spotless.gradle

```diff
@@ -1,3 +1,7 @@
+repositories {
```

Comment:
Because of the gradle issue in these days
https://github.com/diffplug/spotless/issues/1390

It seems resolved in gradle side. I would try to remove the `mavenCentral` and see how the build would be.

Reply:
It has been fixed in gradle side
I removed the mavenCentral assignment and it builds successfully now.
https://github.com/commercetools/commercetools-sync-java/pull/859/commits/ecba302c0acb747eadc013ab8f40a60ff983361b